common/src/main/scala/explore/common/AsterismQueries.scala

```diff
@@ -118,6 +118,9 @@ object AsterismQueries {
       }
     )
 
+  import eu.timepit.refined.auto._
```

Comment:
Maybe imports at  the top?

Reply:
This will be removed shortly when the reload fix is complete.